src/main/java/nl/techiteasy/Techiteasy_part2/Model/Television.java

```diff
@@ -1,7 +1,167 @@
+    public String screenQuality;
+    public boolean smartTv;
+    public boolean wifi;
+    public boolean voiceControl;
+    public boolean hdr;
+    public boolean bleutooth;
+    public boolean ambilight;
+    public int originalStock;
+    public int sold;
+
+    public Television() {
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public String getType() {
+        return type;
+    }
+
+    public String getBrand() {
+        return brand;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public double getPrice() {
+        return price;
+    }
+
+    public double getAvailableSize() {
+        return availableSize;
+    }
+
+    public double getRefreshRate() {
+        return refreshRate;
+    }
+
+    public String getScreentype() {
+        return screentype;
+    }
+
+    public String getScreenQuality() {
+        return screenQuality;
+    }
+
+    public boolean isSmartTv() {
+        return smartTv;
+    }
+
+    public boolean isWifi() {
+        return wifi;
+    }
+
+    public boolean isVoiceControl() {
+        return voiceControl;
+    }
+
+    public boolean isHdr() {
+        return hdr;
+    }
+
+    public boolean isBleutooth() {
+        return bleutooth;
+    }
+
+    public boolean isAmbilight() {
+        return ambilight;
+    }
+
+    public int getOriginalStock() {
+        return originalStock;
+    }
+
+    public int getSold() {
+        return sold;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public void setType(String type) {
+        this.type = type;
+    }
+
+    public void setBrand(String brand) {
+        this.brand = brand;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public void setPrice(double price) {
+        this.price = price;
+    }
+
+    public void setAvailableSize(double availableSize) {
+        this.availableSize = availableSize;
+    }
+
+    public void setRefreshRate(double refreshRate) {
+        this.refreshRate = refreshRate;
+    }
+
+    public void setScreentype(String screentype) {
+        this.screentype = screentype;
+    }
+
+    public void setScreenQuality(String screenQuality) {
+        this.screenQuality = screenQuality;
+    }
+
+    public void setSmartTv(boolean smartTv) {
+        this.smartTv = smartTv;
+    }
+
+    public void setWifi(boolean wifi) {
+        this.wifi = wifi;
+    }
+
+    public void setVoiceControl(boolean voiceControl) {
+        this.voiceControl = voiceControl;
+    }
+
+    public void setHdr(boolean hdr) {
+        this.hdr = hdr;
+    }
+
+    public void setBleutooth(boolean bleutooth) {
+        this.bleutooth = bleutooth;
+    }
+
+    public void setAmbilight(boolean ambilight) {
+        this.ambilight = ambilight;
+    }
+
+    public void setOriginalStock(int originalStock) {
+        this.originalStock = originalStock;
+    }
+
+    public void setSold(int sold) {
+        this.sold = sold;
+    }
```

Comment:
Hi Lex, 

Ziet er goed uit. Twee kleine suggesties over de schrijfwijzes heb ik met je gedeeld in Teams. 

Succes met de studie deze week!